admin/class-automation.php

```diff
@@ -170,6 +178,26 @@ public static function filter__apple_news_exporter_slug( $slug, $post_id ) {
 		return $slug;
 	}
 
+	/**
+	 * A callback function for the apple_news_exporter_title filter.
+	 *
+	 * @param string $title   The title to use.
+	 * @param int    $post_id The post ID associated with the title.
+	 *
+	 * @return string The filtered title value.
+	 */
+	public static function filter__apple_news_exporter_title( $title, $post_id ) {
+		self::$original_title = $title;
+		$rules                = self::get_automation_for_post( $post_id );
+		foreach ( $rules as $rule ) {
+			if ( 'headline.prepend' === ( $rule['field'] ?? '' ) ) {
+				$title = sprintf( '%s %s', $rule['value'] ?? '', self::$original_title );
```

Comment:
Second question: Does the UI allow the user to create multiple of the same type of automation rules? If so, is it common for "last one wins" to be applied, as seems to happen here?

Reply:
Yes, and yes. For example, there might be one behavior if there is a particular category applied, but if there is a specific tag applied, the publisher may want different behavior that is more specific than the generic behavior for the category, so there's the ability to override with "last one wins" logic.